app/controllers/agentAppointment/AppointAgentController.scala

```diff
@@ -75,11 +75,10 @@ class AppointAgentController @Inject()(
       .bindFromRequest()
       .fold(
         hasErrors = errors => {
-          agentsConnector.ownerAgents(request.organisationId) map { ownerAgents =>
+          getAgents(request.organisationId) map { ownerAgents =>
```

Comment:
i think we have agreed to not use infix notation for `map` `flatMap` etc. in production code. It's ok in tests to say `foo shouldBe bar` and not `foo.shouldBe(bar)` but in code we'd like to always consistently use the dot `foo.map(_ => true)`